mbari_wec_all.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 repositories:
-  buoy_msgs:
+  mbari_wec_utils:
```

Comment:
Yeah that's best, I'll update the yaml as well as build instructions

Reply:
Though one concern is there's nothing to build in that, what would `colcon` do in that case? Or there's some way by colcon ignore?

Reply:
I don't think the COLCON_IGNORE is needed. I have that repo in my workspace, and since there's no CMake or package.xml, colcon doesn't do anything with it.

Reply:
Oh okay, removing it